index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
Sorry for not being more clear!

-The plan was to stop adding ST entries to the global performance buffer at `onload`
-Today, RT/UT are collected _forever_ (until RT limit)

-If we keep collecting RT/UT _forever_, then that will be inconsistent with to ST (which may be fine, I don't know)
-If we stop collecting RT/UT at `onload`, then that will break the polling pattern that devs use to be notified about new entries